src/rpc/blockchain.cpp

```diff
@@ -316,6 +324,7 @@ UniValue getblockheader(const UniValue& params, bool fHelp)
             "  \"confirmations\" : n,   (numeric) The number of confirmations, or -1 if the block is not on the main chain\n"
             "  \"height\" : n,          (numeric) The block height or index\n"
             "  \"version\" : n,         (numeric) The block version\n"
+            + strprintf("  \"versionHex\" : \"00000000\", (string) The block version formatted in hexadecimal when version is greater than %d\n", VERSIONBITS_LAST_OLD_BLOCK_VERSION) +
```

Comment:
"when version is greater than %d" looks kind of arbitrary, out of context.
Maybe we should mention BIP9 explicitly in the help.
The reason for this threshold is that BIP9 version bits start to be relevant, right?